### PR TITLE
gotop: 4.1.2 -> 4.1.3

### PR DESCRIPTION
**Pull request by bbjubjub2494**
###### Motivation for this change
https://github.com/xxxserxxx/gotop/releases/tag/v4.1.3

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 160066` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>gotop</li>
  </ul>
</details>

File: pkgs/tools/system/gotop/default.nix [hunk 1]
```diff
@@ -22,6 +22,11 @@ buildGoModule rec {
 
   doCheck = !stdenv.isDarwin;
 
+  postInstall = ''
+    mkdir -p $out/share/man/man1
+    $out/bin/gotop --create-manpage | gzip > $out/share/man/man1/gotop.1.gz
+  '';
```
Comment (fabaff) [inline pkgs/tools/system/gotop/default.nix:28]: Perhaps `installManPage` can help here.

**Review by fabaff — COMMENTED** [review]
- [x] package builds (see https://github.com/NixOS/nixpkgs/pull/160066#issuecomment-1039956968)
- [ ] all depending packages build
- [x] tested on x86_64

```bash
...ew/pr-160066]$ ./results/gotop/bin/gotop -h
Usage of gotop:
        A terminal based graphical activity monitor, inspired by gtop and vtop
Options:
  --remote-name=        Remote: name of remote gotop
  --remote-url=         Remote: URL of remote gotop
[...]
```

Change looks good to me. TUI works.

File: pkgs/tools/system/gotop/default.nix (hunk 1 above)
Comment (lourkeur) [reply pkgs/tools/system/gotop/default.nix:30]: I didn't know that, thanks

**Review by magnetophon — APPROVED** [review]
lgtm

**Review by fabaff — APPROVED** [review]

**Pull request merged**